build.gradle

```diff
@@ -48,20 +48,14 @@ repositories {
 
 dependencies {
     compile gradleApi()
-    compile 'com.android.tools.build:gradle:1.5.0'
-    compile 'com.google.code.gson:gson:2.5'
+    compile 'com.android.tools.build:gradle:2.1.0'
```

Comment:
Exciting!  Thanks for taking the unglamorous job of keeping dependencies up-to-date.

Have you tested this against Android projects that use older build-tools versions?  The major version bump (assuming they follow semver) implies breaking API-level changes; I'm curious if those affect this plugin.